docs/advanced-topics.asciidoc

```diff
@@ -4,5 +4,15 @@
 * <<custom-transactions, Custom Transactions>>
 * <<custom-transaction-name, Initial page load transaction names>>
 
+
+[float]
+[[typescript]]
+==== Using with TypeScript
```

Comment:
This needs to be a level 3 heading without the float tag so it doesn't float to the overview
<img width="649" alt="Screen Shot 2019-12-20 at 10 17 31 AM" src="https://user-images.githubusercontent.com/5618806/71281142-09432400-2312-11ea-8c06-6d17ced11727.png">


```suggestion
=== Using with TypeScript
```